umbraco-cloud/set-up/project-settings/hostname-monitoring.md

```diff
@@ -0,0 +1,108 @@
+---
+description: >
+  A Hostname monitoring is a feature that allows users to track the availability and response times of their websites.
+---
+
+# Hostname Monitoring
+
+Hostname monitoring in Umbraco Cloud allows users to track the availability and response times of their websites. This feature helps ensure optimal website performance and alerts users to potential downtime or performance issues. Users can add, edit, and view hostname monitors through the Umbraco Cloud Portal. A historical overview of uptime is available in the **Ping Results** section.
+
+Hostname monitoring can be accessed under **Insights > Hostname Monitoring** on your project in the Umbraco Cloud Portal.
+
+## Hostname Monitor Configuration
+
+- **Hostname**: The domain or subdomain to monitor (e.g., `https://example.com` or `http://web.example.com`). Users can also monitor hostnames outside of Umbraco Cloud, such as static frontend websites hosted elsewhere for Heartcore projects.
+- **Frequency**: The interval at which the hostname is checked (e.g., every X minutes). Valid values: 1, 5, 10, 15, 30, 60, 360, 720, 1440 minutes.
+- **Locations**: Users can select multiple locations to ping the web application from. Supported regions include Europe, the US, Australia, and the UK.
+- **Max Response Time**: The maximum acceptable response time in seconds. Must be greater than 0.
+- **Expected Status Code**: The HTTP status code that indicates a successful response (e.g., `200`, `301`, `404`). Any status code can be monitored, and this value determines whether the UI marks the response as a failure or success.
```

Comment:
Should we add more information on the status codes examples?

e.g., 200 - OK, 301 - Moved Permanently, 404 - Not Found

Reply:
Anyone can google status codes, we can let developer.mozilla.org be the authority.